core/arch/arm/tee/arch_svc.c

```diff
@@ -241,6 +242,9 @@ void __weak tee_svc_handler(struct thread_svc_regs *regs)
 	}
 }
 
+/*  Max size of the structure storing the  */
+#define TA_CONTEXT_MAX_SIZE		sizeof(struct thread_abort_regs)
```

Comment:
Why not use the exact size of what is pushed by `utee_panic()`? That is: `14*4` for 32-bit TAs and `2*8` for 64-bit TAs.